src/app/components/landing-page/landing-page.component.ts

```diff
@@ -41,10 +45,16 @@ export class LandingPageComponent implements OnInit {
 
  }
 
+/**
+ * DUPLICATED
```

Comment:
Why do multiple comments simply say duplicated?

Reply:
Because that is duplicated code, which I already commented elsewhere. I can copy the other comments in if you prefer, but I thought it would be better if it were clear to people reading that the code is duplicated.